stream-chat-android-ui-components/src/main/res/values-en/strings_message_list_header.xml

```diff
@@ -6,8 +6,8 @@
 
     <!-- Header Subtitle -->
     <string name="stream_ui_message_list_header_thread_subtitle">with %1$s</string>
-    <string name="stream_ui_message_list_header_disconnected">Searching for network</string>
-    <string name="stream_ui_message_list_header_offline">Offline…</string>
+    <string name="stream_ui_message_list_header_disconnected">Waiting for network</string>
+    <string name="stream_ui_message_list_header_offline">Disconnected</string>
```

Comment:
The name of the string and the content (when compared to the string above) might be misleading, but I wanted to keep the names to avoid breaking changes. cc @adasiewiczr